notebooks/official/tensorboard/tensorboard_vertex_pipeline.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Vertex Pipeline -&gt; Vertex AI Pipeline</p><p> </p><p>You can click -&gt; click</p><p> </p><p>, it will redirect -&gt; and it takes you</p><p> </p><p>And in this page you can click -&gt; In the custom jobs page, click</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>